tests/bot/exts/filters/test_token_remover.py

```diff
@@ -248,18 +294,84 @@ def test_format_log_message(self, log_message):
             hmac="x" * len(token.hmac),
         )
 
+    @autospec("bot.exts.filters.token_remover", "UNKNOWN_USER_LOG_MESSAGE")
+    def test_format_userid_log_message_unknown(self, unknown_user_log_message):
+        """
+        Should correctly format the user ID portion of the log message when the user ID is
+         not found in the server.
```

Comment:
Summaries should never be more than one line long. Also, don't put leading space on subsequent lines in docstrings.